doc/developer-guide/internal-libraries/MemArena.en.rst

```diff
@@ -76,14 +76,14 @@ to not be an issue, or there must be a provision for some sort of garbage collec
 
 Generally |MemArena| is not as useful for classes that allocate their own internal memory
 (such as :code:`std::string` or :code:`std::vector`), which includes most container classes. One
-container class that can be easily used is :class:`IntrusiveDList` because the links are in the
+container class that can be easily used is :code:`swoc::IntrusiveDList` because the links are in the
 instance and therefore also in the arena.
 
 Objects created in the arena must not have :code:`delete` called on them as this will corrupt
 memory, usually leading to an immediate crash. The memory for the instance will be released when the
 arena is destroyed. The destructor can be called if needed but in general if a destructor is needed
 it is probably not a class that should be constructed in the arena. Looking at
-:class:`IntrusiveDList` again for an example, if this is used to link objects in the arena, there is
+:code`:`swoc::IntrusiveDList` again for an example, if this is used to link objects in the arena, there is
```

Comment:
For some reason, Sphinx only likes this the first time it's used:  https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/2470/Sphinx_20Docs/

Reply:
The underlying problem is the `class` data is populated by other document files, in this particular case by "intrusive_dlist.en.rst" which no longer exists.

Reply:
The change doesn' t look any better to me:  https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/2472/Sphinx_20Docs/developer-guide/internal-libraries/MemArena.en.html#memarena .  But anyway.

Reply:
The curse of Sphinx is inescapable.